ResearchKit/ActiveTasks/ORKTrailmakingContentView.h

```diff
@@ -0,0 +1,53 @@
+/*
+ Copyright (c) 2016, Motus Design Group Inc. All rights reserved.
```

Comment:
No. The submission rules are that for any significant code change, you should add your name (or your company's name) to the copyright.